source/common/upstream/upstream_impl.h

```diff
@@ -503,6 +503,7 @@ class ClusterImplBase : public Cluster, protected Logger::Loggable<Logger::Id::u
 
 protected:
   PrioritySetImpl priority_set_;
+  std::unordered_map<std::string, Upstream::HostSharedPtr> all_hosts_;
```

Comment:
Is this used in the base class? Should this be part of  `BaseDynamicClusterImpl` ?